locales/en/plugin__mcg-ms-console.json

```diff
@@ -78,60 +79,68 @@
   "Access Key Field": "Access Key Field",
   "Switch to Secret": "Switch to Secret",
   "Secret Key Field": "Secret Key Field",
-  "ObjectBucketClaim Name": "ObjectBucketClaim Name",
-  "my-object-bucket": "my-object-bucket",
+  "ObjectBucketClaim name": "Object bucket claim name",
+  "my-object-bucket-claim": "my-object-bucket-claim",
   "If not provided a generic name will be generated.": "If not provided a generic name will be generated.",
-  "StorageClass": "StorageClass",
+  "StorageClass": "Storage class",
   "Defines the object-store service and the bucket provisioner.": "Defines the object-store service and the bucket provisioner.",
-  "Bucket Policy": "Bucket Policy",
-  "Create ObjectBucketClaim": "Create ObjectBucketClaim",
+  "Bucket policy": "Bucket policy",
+  "Create new ObjectBucketClaim": "Create new object bucket claim",
+  "Create ObjectBucketClaim": "Create object bucket claim",
```

Comment:
keys and values can never be different... there is some issue... try `yarn i18n`